lib/guest_installation_and_configuration_base.pm

```diff
@@ -2572,12 +2572,9 @@ sub start_guest_installation {
     my $_guest_installation_log = "$common_log_folder/$self->{guest_name}/$self->{guest_name}" . "_installation_log_" . $_start_installation_timestamp;
     assert_script_run("touch $_guest_installation_log && chmod 777 $_guest_installation_log");
     # Dry run always timeout when downloading initrd from download.opensuse.org in O3
-    my $ret = script_run("set -o pipefail; $self->{virt_install_command_line_dryrun} 2>&1 | tee -a $_guest_installation_dryrun_log", timeout => 600 / get_var('TIMEOUT_SCALE', 1), die_on_timeout => 0);
+    my $ret = script_run("set -o pipefail; timeout 580 $self->{virt_install_command_line_dryrun} 2>&1 | tee -a $_guest_installation_dryrun_log", timeout => 600 / get_var('TIMEOUT_SCALE', 1));
```

Comment:
@waynechen55 Is `timeout 580` or `timeout 580/get_var('TIMEOUT_SCALE', 1)`  more approriate to be inline with `timeout => xxx` at here?

Reply:
```suggestion
    my $ret = script_run("set -o pipefail; timeout " . 580 / get_var('TIMEOUT_SCALE', 1) . "$self->{virt_install_command_line_dryrun} 2>&1 | tee -a $_guest_installation_dryrun_log", timeout => 600 / get_var('TIMEOUT_SCALE', 1));
```

would probably be more consistent with the current timeout. But I would just go with

```suggestion
    my $ret = script_run("set -o pipefail; timeout 580 $self->{virt_install_command_line_dryrun} 2>&1 | tee -a $_guest_installation_dryrun_log", timeout => 600);
```

I see not much point in waiting less and ignoring the result if TIMEOUT_SCALE is set to a non-standard value.

Reply:
Please see my updated version.